packages/components/src/clipboard-button/README.md

```diff
@@ -24,3 +28,39 @@ const MyClipboardButton = () => {
 	);
 };
 ```
+
+## Props
+
+The component accepts the following props:
+
+### className
+
+The class that will be added to the classes of the underlying `<Button>` component.
+
+- Type: `string`
+- Required: no
+
+### text
+
+The text that will be copied to the clipboard.
+
+- Type: `string`
+- Required: no
+
+### onCopy
+
+The function that will be called when the text is copied.
+
+-- Type: `() => void`
+-- Required: no
```

Comment:
From a quick look at the code, it seems like `onCopy` would be required

Reply:
Nice catch! PR updated.